org.mixedrealitytoolkit.uxcore/Slider/SliderSounds.cs

```diff
@@ -151,13 +136,9 @@ public AudioSource PassNotchAudioSource
         /// </summary> 
         private void Start()
         {
-            if (grabReleaseAudioSource == null)
-            {
-                grabReleaseAudioSource = gameObject.AddComponent<AudioSource>();
-            }
-            if (passNotchAudioSource == null)
+            if (audioSource == null)
             {
-                passNotchAudioSource = gameObject.AddComponent<AudioSource>();
+                audioSource = gameObject.GetComponent<AudioSource>();
```

Comment:
We should still create a AudioSource if missing, to maintain old behavior.   It's feasible apps could be using their own slider prefab, relying on the auto creation of the audio source.

```
if (audioSource == null)
{
    audioSource = gameObject.GetComponent<AudioSource>();
    if (audioSource == null)
    {
        audioSource = gameObject.AddComponent<AudioSource>();
   }
}

Reply:
will make the change.

Reply:
Might be worth checking for the field being null _and_ checking `TryGetComponent` in the `if`, then adding if it's not found through either of those paths.

Reply:
that is the plan :)

Reply:
change made.